app/javascript/controllers/groups_controller.js

```
@@ -76,39 +223,49 @@ export default class extends Controller {
-            let pastedEmails = '';
-            if (window.clipboardData && window.clipboardData.getData) {
-                pastedEmails = window.clipboardData.getData('Text');
-            } else if (e.clipboardData && e.clipboardData.getData) {
-                pastedEmails = e.clipboardData.getData('text/plain');
-            }
+        document
+            .querySelector('.select2-selection input')
+            .addEventListener('paste', (e) => {
+                e.preventDefault();
+                let pastedEmails = '';
+                if (window.clipboardData && window.clipboardData.getData) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)